engine/include/cubos/engine/gl/deferred/renderer.hpp

```diff
@@ -33,13 +33,20 @@ namespace cubos::engine::gl::deferred
         virtual RendererGrid upload(const core::gl::Grid& grid) override;
         virtual void setPalette(const core::gl::Palette& palette) override;
 
+        bool isSSAOEnabled() const;
+        void setSSAOEnabled(bool ssaoEnabled);
```

Comment:
Maybe instead of getting/setting data from here, we should we just fetch it directly from the Settings global object.
This way we avoid dynamic casts later on.

Reply:
We should recheck the value from the settings onResize

Reply:
Fixed. Also onResize was already implemented :relieved: